lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ScopeSet } from "./ScopeSet";
+import { StringUtils } from "./../utils/StringUtils";
+import { ClientConfigurationError } from "./../error/ClientConfigurationError";
+import { PromptValue, CodeChallengeMethodValues } from "./../utils/Constants";
+
+/**
+ * Validates server consumable params from the "request" objects
+ */
+export class RequestValidator {
+
+    /**
+     * Utility to validate scopes passed by the user in the request
+     */
+    static validateAndGenerateScopes(scopes: Array<string>, clientId: string): Array<string> {
```

Comment:
This seems like a pointless function. You should use the ScopeSet object throughout the code until the request is made.

Reply:
Switched over to using ScopeSet